doc/user/content/cli/_index.md

```diff
@@ -127,11 +128,25 @@ the compaction window.
 
 ### Logging
 
+#### Log file
+
+The `--log-file` option specifies the path to a file in which Materialize will
+write its [log messages](/ops/monitoring#logging). The special value `stderr`
+specifies the standard error stream.
+
```

Comment:
I don't understand where/how you specify `stderr`. Ignore me if this is obvious to a developer. 

It's probably worth defining what the standard error stream is and where you would find it, or directing people to the doc on that if it exists.

Reply:
Reworded to make it a bit more clear that `stderr` is a special value that means "write to the standard error stream." I don't think we need to define the standard error stream here though; that's Unix 101.